ddocs/medic-client/views/doc_summaries_by_id/map.js

```diff
@@ -56,6 +56,7 @@ function(doc) {
       read: doc.read,
       valid: !doc.errors || !doc.errors.length,
       verified: doc.verified,
+      verified_valid: doc.verified_valid,
```

Comment:
I was hoping we could use one field for all three states, eg: `verified` could be `undefined`, `true`, or `false` representing "unverified", "correct" and "has errors" respectively. This would make the code a little simpler but may have complications for backwards compatibility. Would this work and be better?

Reply:
I chose this option so it would be backwards compatible and it would mean that this PR doesn't break existing functionality. 
I know that there are 2 more issues which branch from this (#4577 and #4575), which would potentially fix the compatibility, but it would mean that this PR leaves the app in a broken state. 

I was also concerned about whether, if not backwards compatible, the PR should include a migration and what this migration should do (are we considering previously verified reports as being valid or remove their verification status altogether?).  
The other concern was analytics - whether or not changing the meaning of the `verified` property would require effort on their part.

Reply:
We certainly don't want to do migrations unless absolutely necessary. As far as I know nobody is using verification currently, which would imply backwards compatibility isn't required. Are docs created as `verified: false` or `verified: undefined` do you know? Because we don't want existing docs to go from "unknown" to "has errors"...

@sglangevin Do you know of anyone using the "verify" functionality in production?

Reply:
As far as I can tell, new reports do not have a `verified` property. `verified == false` happens only when someone un-verifies a previously verified report.

Reply:
Ok so as long as nobody is using the feature, then backwards compatibility will just work if `undefined`, `false`, and `true` equate to "not yet verified", "has errors", and "verified valid" respectively.

Reply:
As far as I know, no one is actively using this feature. @dianabarsan is correct, if you want to get `verified == false` you have to verify and then un-verify a report. So I think @garethbowen's proposal will work fine. Even if people were using it, they were only looking at `verified == true` and we aren't changing that part, so I think we're good.